actors.md

```diff
@@ -477,6 +478,16 @@ func CommitSector(sectorID SectorID, commD, commR, commRStar []byte, proof SealP
 		miner.ProvingPeriodEnd = chain.Now() + ProvingPeriodDuration(miner.SectorSize)
 	}
 }
+
+func collateralForPower(power BytesAmount) TokenAmount {
```

Comment:
Nit: Is the lower case initial character intended to be meaningful? It will be quite opaque to anyone not familiar with Go and it looks like the standard here is to uppercase all function/method names everywhere. Visibility isn't relevant to the spec.

Reply:
Nope... that was just me being javascript developer and naming variables like I'm in JSland